src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderViewMenu.vue

```diff
@@ -57,6 +61,16 @@ export default {
 				label: this.$t('calendar', 'Month'),
 			}]
 		},
+		shortKeyConf() {
+			return {
+				timeGridDay: ['d'],
+				timeGridDay_Num: [1],
```

Comment:
Seems to be an issue with `vue-shortkey`, but only numpad number keys work, not the combination <kbd>Maj</kbd> + <kbd>&</kbd> on my Azerty keyboard (event code `Digit1` vs `Numpad1`).

Reply:
This seems to be an issue specific to the AZERTY keyboard :/ 

On my QWERTY keyboard, both Numpad and number keys work just fine.